raiden/raiden_service.py

```diff
@@ -444,6 +444,22 @@ def expect_exchange(
         asset_manager = self.raiden.get_manager_by_asset_address(from_asset)
         asset_manager.transfermanager.exchanges[ExchangeKey(from_asset, from_amount)] = exchange
 
+    def get_channel_list(self, asset_address=None, partner_address=None):
```

Comment:
Can you add a detailed docstring description following the [google style of comment coding](http://google.github.io/styleguide/pyguide.html?showone=Comments#Comments) here?